.github/workflows/dependency-review.yml

```diff
@@ -0,0 +1,14 @@
+name: "Dependency Review"
```

Comment:
What's the difference between dependency review and security scan?

Reply:
Dependency review is a SCA checks which does two things:
- it ensures the dependencies we use satisfies the legal framework we have for open source dependencies.
- it ensures the dependencies we use are vulnerability free

Security Scan is a SAST check: it ensures (to an extent) that the code we wrote is not exploitable.